src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -214,9 +214,7 @@ def apply_quantization_status(model: Module, status: QuantizationStatus):
     :param status: status to update the module to
     """
 
-    current_status = infer_quantization_status(model)
```

Comment:
Theoretically, you could replace this with `infer_quantization_status(module)`, ie `getattr(module, "quantization_status", None)`

Reply:
I removed `infer_quantization_status` entirely because it is only used here and we are updating this to no longer care about current status. The application will occur regardless, which is necessary if we have multiple `QuantizationMixin`s in a single recipe. With these changes, it no longer makes sense to infer quantization status of a model, and if we do we should just `getattr(module, "quantization_status", None)` as you suggest